defaults/main.yml

```diff
@@ -419,7 +443,8 @@ postgresql_cursor_tuple_fraction:      0.1          # range 0.0-1.0
 postgresql_from_collapse_limit:        8
 postgresql_join_collapse_limit:        8            # 1 disables collapsing of explicit
 postgresql_force_parallel_mode:        off          # on, off, regress (>= 9.6)
-postgresql_jit:                        off          # (>= 11)
+postgresql_jit:                        on           # (>= 11: off, 12: on)
```

Comment:
New default for v12

Reply:
The old default version is now set in vars/postgresql_11.yml... so behaviour shouldn't change for old versions.